lms/urls.py

```diff
@@ -1055,3 +1055,8 @@
 urlpatterns += [
     path('api/notifications/', include('openedx.core.djangoapps.notifications.urls')),
 ]
+
+# Content tagging
+urlpatterns += [
+    path('api/tagging/', include(('openedx.features.content_tagging.urls', 'content_tagging'), namespace='content_tagging')),
+]
```

Comment:
My tests are running from lms env.
```
# pytest openedx/features/content_tagging/rest_api

platform linux -- Python 3.8.17, pytest-7.3.2, pluggy-1.0.0
Using --randomly-seed=3629427016
django: settings: lms.envs.test (from ini)
rootdir: /home/humano/Projetos/opencraft/edx-platform
configfile: setup.cfg
plugins: randomly-3.12.0, Faker-18.10.1, json-report-1.5.0, attrib-0.1.3, django-4.5.2, cov-4.1.0, xdist-3.3.1, metadata-1.8.0, anyio-3.7.0
collected 9 items
```

I didn't found where I can change that.

Reply:
Ah..  Probably easiest to just add the tagging API URLs to lms.urls too, and so your tests will run in both the CMS and LMS environments.

We're not using tagging in the LMS right now, but we will post-MVP. If we were never going to use them in the LMS, I'd introduce you to the fun of wrapping your tests in [`@skip_unless_cms`](https://github.com/openedx/edx-platform/blob/41ec0852e93b11087bc7d6ebcbb78241d56b90ab/openedx/core/djangolib/testing/utils.py#L244) and show you this:

```
DJANGO_SETTINGS_MODULE=cms.envs.test pytest -vv openedx/features/content_tagging/
```